package.json

```diff
@@ -19,7 +19,7 @@
     "prepush": "yarn git-hook && yarn snyk test",
     "start": "node app.js",
     "start-watch": "nodemon app.js | ./node_modules/bunyan/bin/bunyan & nodemon -e scss -x yarn build-css -- compact",
-    "test": "NODE_ENV=test mocha --recursive test",
+    "test": "NODE_ENV=test mocha --recursive test && snyk test",
```

Comment:
Does `&& snyk test` need adding? It had been removed previously for the reason of not wishing to run it locally and have the GitHub integration pick up the running.